chainbench/profile/evm/heavy.py

```diff
@@ -33,31 +34,33 @@ def eth_get_logs_task(self):
         self.make_call(
             name="eth_get_logs",
             method="eth_getLogs",
-            params=self._get_logs_params_factory(),
+            params=self._get_logs_params_factory(get_rng()),
         ),
 
     @task
     def eth_get_blocks_receipts_task(self):
         self.make_call(
             name="eth_get_block_receipts",
             method="eth_getBlockReceipts",
-            params=[self.test_data.get_random_block_number_hex()],
+            params=[hex(self.test_data.get_random_block_number(get_rng()))],
         )
 
     @task
     def trace_replay_transaction_task(self):
         self.make_call(
             name="trace_replay_transaction",
             method="trace_replayTransaction",
-            params=self._trace_replay_transaction_by_hash_params_factory(),
+            params=self._trace_replay_transaction_params_factory(get_rng()),
         )
 
     @task
     def trace_replay_block_transactions_task(self):
         self.make_call(
             name="trace_replay_block_transactions",
             method="trace_replayBlockTransactions",
-            params=self._trace_replay_block_transaction_by_block_number_params_factory(),  # noqa E501
+            params=self._trace_replay_block_transaction_params_factory(
+                get_rng()
+            ),  # noqa E501
```

Comment:
why `noqa` here?

Reply:
hmm. I think it used to all be on one line, and the name was longer, so it needed the #noqa. the name has been shortened so I just put it back on one line and removed the #noqa. thanks for spotting this!